virttest/test_setup.py

```diff
@@ -1425,7 +1413,7 @@ def sr_iov_setup(self):
                               logging.info)
         if ARCH != 'ppc64le':
             kvm_re_probe = True
-            o = process.system_output("dmesg")
+            o = process.system_output("dmesg", verbose=False)
```

Comment:
commit msg says, "Remove dmesg info in log" but only verbose is turned off, lets have more clear msg.

Reply:
Thanks for your review. Updated. How about now ?

Reply:
@balamuruhans  Could you please help review again, thanks!

Reply:
please use proper variable name instead of `o`